internal/sirius/add_note.go

```diff
@@ -0,0 +1,60 @@
+package sirius
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+type addNoteRequest struct {
+	Title       string `json:"name"`
+	Note        string `json:"description"`
+	UserId		int    `json:"createdById"`
+	NoteType	string    `json:"noteType"`
```

Comment:
Can you run `go fmt`? I think some indents are not in alignment.

Reply:
go fmt didn't seem to apply anything, do I need to do something to set it up? Otherwise I used the IDE to fix the indexing 👍